02Lesson/02Project/css/style.css

```diff
@@ -4,8 +4,16 @@ body {
 }
 
 header, nav, main, section, aside, footer {
+  margin: 0;
+  padding: 0;
   padding: 5px;
   margin-top: 10px;
+
+}
+
+.ArticleHeader {

```

Comment:
I would use lowercase class names. If you are using it consistently, this is fine, but later you might want to add custom JS plugins which almost always use lowercase naming convention or something like `.module-classname` for longer words. Then you have mixed naming conventions in your code which will get confusing.